plugins/modules/azure_rm_adapplication_info.py

```diff
@@ -179,9 +190,25 @@ async def get_applications(self, sub_filters):
                     filter=(' and '.join(sub_filters)),
                 ),
             )
-            return await self._client.applications.get(request_configuration=request_configuration)
+            applications = await self._client.applications.get(request_configuration=request_configuration)
+            return applications.value
         else:
-            return await self._client.applications.get()
+            applications_list = []
+            applications = await self._client.applications.get()
+            for app in applications.value:
+                applications_list.append(app)
+            
```

Comment:
blank line contains whitespace
```suggestion

```

Reply:
I've removed the whitespace as requested.